### PR TITLE
✨ Update for the Portal ETL

### LINKED ISSUE #10: Add Experiment Strategy
Currently, the experiment strategy field is not properly populated. Fix this. This request is per https://www.notion.so/ferlab/cc83f8e35fad437a923c9aa2aeae5c33?v=70690137fddb401d910043aead0bb646.

### PR DESCRIPTION
**Pull request by liberaliscomputing**
This PR adds:
- experiment strategy (See #10);
- tumor descriptor (See #11);
- proband status (See #12); and
- updates to Specimen (See #13)

It closes #10, #11, #12, and #13.

### CODE REVIEW

File: kf_task_fhir_etl/target_api_plugins/entity_builders/proband_status.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+"""
+Builds FHIR Observation resources (https://www.hl7.org/fhir/observation.html)
+from rows of tabular participant broband status data.
```
Comment (znatty22) [inline kf_task_fhir_etl/target_api_plugins/entity_builders/proband_status.py:3]:
```suggestion
from rows of tabular participant proband status data.
```

File: kf_task_fhir_etl/target_api_plugins/entity_builders/proband_status.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            "resourceType": cls.api_path,
+            "id": get_target_id_from_record(cls, record),
+            "meta": {
+                "profile": [f"http://hl7.org/fhir/StructureDefinition/{cls.api_path}"],
+                "tag": [{"code": study_id}],
+            },
+            "identifier": [
+                {
+                    "use": "official",
+                    "system": "https://kf-api-dataservice.kidsfirstdrc.org/participants?is_proband=",
```
Comment (znatty22) [inline kf_task_fhir_etl/target_api_plugins/entity_builders/proband_status.py:62]:
```suggestion
                    "system": "https://kf-api-dataservice.kidsfirstdrc.org/participants?is_proband",
```

File: kf_task_fhir_etl/target_api_plugins/entity_builders/histopathology.py [hunk 3]
```diff
@@ -15,6 +16,14 @@
 # http://hl7.org/fhir/ValueSet/observation-status
 status_code = "final"
 
+missing_data_values = {
```
Comment (znatty22) [inline kf_task_fhir_etl/target_api_plugins/entity_builders/histopathology.py:19]: Should these be in a common module somewhere?

File: kf_task_fhir_etl/target_api_plugins/entity_builders/proband_status.py (hunk 2 above)
Comment (znatty22) [reply kf_task_fhir_etl/target_api_plugins/entity_builders/proband_status.py:62]: I'm leaving one comment here but it applies to all the occurrences of this observation. I'm wondering why these `system` fields that have a url + query parameter are appended with the `=` sign?

**Review by znatty22 — DISMISSED** [review]
Looks pretty good just left some minor comments/questions

File: kf_task_fhir_etl/target_api_plugins/entity_builders/proband_status.py (hunk 2 above)
Comment (liberaliscomputing) [reply kf_task_fhir_etl/target_api_plugins/entity_builders/proband_status.py:62]: Let me create another ticket to discuss this.

**Pull request merged**